frontend/packages/pipelines-plugin/src/components/pipelines/utils/pipeline-operator.ts

```diff
@@ -101,3 +102,11 @@ export const useIsTektonV1VersionPresent = (setFeatureFlag: SetFeatureFlag) => {
   const isTektonV1VersionPresent = operatorVersion?.major === 1 && operatorVersion?.minor >= 11;
   setFeatureFlag(FLAG_TEKTON_V1_ENABLED, isTektonV1VersionPresent);
 };
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const useIsPipelineOperatorVersion_1_16 = (setFeatureFlag: SetFeatureFlag) => {
+  const [activeNamespace] = useActiveNamespace();
+  const operatorVersion = usePipelineOperatorVersion(activeNamespace);
+  const IS_PIPELINE_OPERATOR_VERSION_1_16 =
```

Comment:
should this flag be named `is_pipieline_operator_1_16_or_newer`? 

also should a ticket be filed to remove the extra versions checks for when the 1.15 pipelines operator goes EOL?

Reply:
Created issue - https://issues.redhat.com/browse/OCPBUGS-43075 for version check removal and updated PR for flag naming changes